tests/0.8.1/invalid.json

```diff
@@ -0,0 +1,842 @@
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkgaAaVSzE9XqanrbFD1J9RGTFcWU7N9BPgjSWm17pk1mB",
+        "aud": "did:key:z6MkrjzAyNQV2HmFRiLhNoiTW7RZ76JUTnuHtfsTN1RK6ZPT",
+        "exp": 5435042552,
+        "att": [],
+        "prf": [
+          "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtueFZhWFN4VnRjZWR6Wm45aFVUTkxmYk5MUHhyaUVyWGRXODZ2cm5mVVN4QyIsImF1ZCI6ImRpZDprZXk6ejZNa2dhQWFWU3pFOVhxYW5yYkZEMUo5UkdURmNXVTdOOUJQZ2pTV20xN3BrMW1CIiwiZXhwIjo1NDM1MDQyNTUyLCJhdHQiOltdLCJwcmYiOltdfQ.5K4narVH1WEoS_x3eP_sbYIljlGOdsTg-dM-_x3W-Q09NQOX75ZdJTh4kZ4ff4Gx81nb9ZjYOxrtb6wCS4ABDA"
+        ]
+      },
+      "validationErrors": [
+        "expWitnessTimeBoundExceeded"
+      ]
+    }
+  },
+  {
+    "comment": "Witnesses are not ready to be used before the delegated UCAN",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtnYUFhVlN6RTlYcWFucmJGRDFKOVJHVEZjV1U3TjlCUGdqU1dtMTdwazFtQiIsImF1ZCI6ImRpZDprZXk6ejZNa3JqekF5TlFWMkhtRlJpTGhOb2lUVzdSWjc2SlVUbnVIdGZzVE4xUks2WlBUIiwibmJmIjoxNjQ4MjE2OTUyLCJleHAiOjU0MzUwNDI1NTIsImF0dCI6W10sInByZiI6WyJleUpoYkdjaU9pSkZaRVJUUVNJc0luUjVjQ0k2SWtwWFZDSXNJblZqZGlJNklqQXVPQzR4SW4wLmV5SnBjM01pT2lKa2FXUTZhMlY1T25vMlRXdHlhSGMyY1V0RFJsQkVWbFV4ZW1kMU1VNXlOSEkyV2t4VGNESmtTamhUVWxGa1pUWk1hVmROUzBOa1JDSXNJbUYxWkNJNkltUnBaRHByWlhrNmVqWk5hMmRoUVdGV1UzcEZPVmh4WVc1eVlrWkVNVW81VWtkVVJtTlhWVGRPT1VKUVoycFRWMjB4TjNCck1XMUNJaXdpYm1KbUlqbzBPREF6T0Rrd05UVXlMQ0psZUhBaU9qVTBNelV3TkRJMU5USXNJbUYwZENJNlcxMHNJbkJ5WmlJNlcxMTkuLXMtbEMwbkpNWmswemxualNhbG80SWtRX2ZzbHpoeXU0NjRHcVFPaEl5bzduMEI1dXZaVGFGT0laOFpzYWxlZDh5REFLZTkzYnRYd2xqWEppRWdpQUEiXX0.VDG8KaYCDp5FQ_QyrR1bMndqHATXNnykDa0tDGXRyCQZGy7tthazQMnt1Vnj98v3iT7xlbv8G8MN4kXKEs3qAQ",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkgaAaVSzE9XqanrbFD1J9RGTFcWU7N9BPgjSWm17pk1mB",
+        "aud": "did:key:z6MkrjzAyNQV2HmFRiLhNoiTW7RZ76JUTnuHtfsTN1RK6ZPT",
+        "nbf": 1648216952,
+        "exp": 5435042552,
+        "att": [],
+        "prf": [
+          "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtyaHc2cUtDRlBEVlUxemd1MU5yNHI2WkxTcDJkSjhTUlFkZTZMaVdNS0NkRCIsImF1ZCI6ImRpZDprZXk6ejZNa2dhQWFWU3pFOVhxYW5yYkZEMUo5UkdURmNXVTdOOUJQZ2pTV20xN3BrMW1CIiwibmJmIjo0ODAzODkwNTUyLCJleHAiOjU0MzUwNDI1NTIsImF0dCI6W10sInByZiI6W119.-s-lC0nJMZk0zlnjSalo4IkQ_fslzhyu464GqQOhIyo7n0B5uvZTaFOIZ8Zsaled8yDAKe93btXwljXJiEgiAA"
+        ]
+      },
+      "validationErrors": [
+        "expWitnessTimeBoundExceeded"
+      ]
+    }
+  },
+  {
+    "comment": "Witness issuer audience did does not align with delegated issuer did",
```

Comment:
It may be easier to distinguish the decentralized identifiers in this comment if we change "did" to "DID".